TWINLocal/Mutex.cpp

```
@@ -0,0 +1,28 @@
+#include "Mutex.h"
+
+Mutex::Mutex() : m_ghMutex(CreateMutex(NULL, FALSE, TEXT("TechnicianMutex")))
+{
+    // left blank intentionally
+}
+
+
+Mutex::~Mutex()
+{
+    CloseHandle(m_ghMutex);
+}
+
+BOOL Mutex::isMutexTaken() const 
```

instead of returning bool, you can change it to isValid or verifyMutex and not return anything, but throw an exception if its not valid.